src/java.base/share/classes/java/lang/invoke/MemberName.java

```diff
@@ -690,7 +654,7 @@ public MemberName(Field fld, boolean makeSetter) {
         assert(isResolved() && this.clazz != null);
         this.name = fld.getName();
         this.type = fld.getType();
-        assert((REF_putStatic - REF_getStatic) == (REF_putField - REF_getField));
+        // assert((REF_putStatic - REF_getStatic) == (REF_putField - REF_getField));
```

Comment:
Intellij says this is always true. Maybe it could be put in the `static {}` block instead?

Also, I think this should just be removed if it's not needed. (instead of commented out)

Reply:
Will fix by extracting to a `static` block above